### ISSUE TITLE
Incorrect styling in explore skeleton example

### ISSUE THREAD

**Issue by litteredwitherrors** [microsoft/fast#3985]
**Describe the bug; what happened?**

The component with a `shape="circle"` attribute applied has a `border-radius` applied to it in the inline-styles forcing it to display as a rectangle. Fix instructions below:

_Incorrect_
```html
<div>
    <div class="card-placeholder">
        <fast-skeleton style="border-radius: 4px; width: 50px; height: 50px;" shape="circle"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; width: 75px; height: 30px; margin-top: 20px; margin-bottom: 10px;" shape="rect"></fast-skeleton>
    </div>
    <style>
        .card-placeholder {
            background-color: #fff;
            padding: 20px;
            width: 500px;
            border-radius: 4px;
        }
    </style>
</div>
```

_Correct_
```html
<div>
    <div class="card-placeholder">
        <fast-skeleton style="width: 50px; height: 50px;" shape="circle"></fast-skeleton> /*border-radius removed*/
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; margin-top: 10px; height: 10px;" shape="rect"></fast-skeleton>
        <fast-skeleton style="border-radius: 4px; width: 75px; height: 30px; margin-top: 20px; margin-bottom: 10px;" shape="rect"></fast-skeleton>
    </div>
    <style>
        .card-placeholder {
            background-color: #fff;
            padding: 20px;
            width: 500px;
            border-radius: 4px;
        }
    </style>
</div>
```

**What are the steps to reproduce the issue?**
1. Go to: https://explore.fast.design/components/fast-skeleton
2. See example

**What behavior did you expect?**
The component should be a circle.

**Comment by janechu** [comment]
This can be fixed after #3930 as that issue is slated for this week.

**Comment by janechu** [comment]
This was fixed in https://github.com/microsoft/fast/pull/4005/files#diff-4ac42f1437a58874fab6ca9e2b0f631035c5a377c972de907cf006eed878cc6b
